test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -362,6 +368,41 @@ UCS_TEST_SKIP_COND_P(test_ib_md, mt_fail,
     }
 }
 
+UCS_TEST_SKIP_COND_P(test_ib_md, mem_window,
+                     !check_invalidate_support(UCT_MD_MEM_ACCESS_RMA))
+{
+    unsigned flags = UCT_MD_FLAG_INVALIDATE_RMA | UCT_MD_FLAG_INVALIDATE_AMO;
+    std::vector<uint8_t> buffer(1024);
+    uct_mem_h base;
+    EXPECT_UCS_OK(reg_mem(UCT_MD_MEM_ACCESS_RMA, buffer.data(), buffer.size(),
+                          &base));
+
+    /* Test case 1: creating MW from memh before mkey_pack */
+    uct_mem_h mw1 = reg_mem_window(base);
+
+    /* Test case 2: creating MW from memh after mkey_pack */
+    std::vector<uint8_t> base_rkey1 = mkey_pack(base, flags);
```

Comment:
do we need invalidation on first mkey pack of base?

Reply:
do we need test with packing without invalidation first?

Reply:
In terms of UCT invalidation would mean that we destroy derived memh. Currently I don't have tests with invalidation, actually it's a good point to add them.
Or I misunderstood your comment